src/libraries/System.Runtime/tests/System/HalfTests.cs

```diff
@@ -281,6 +281,10 @@ public static IEnumerable<object[]> CompareTo_TestData()
             yield return new object[] { Half.NaN, Half.NaN, 0 };
             yield return new object[] { Half.NaN, UInt16BitsToHalf(0x0000), -1 };
             yield return new object[] { Half.MaxValue, null, 1 };
+            yield return new object[] { Half.MinValue, Half.NegativeInfinity, 1 };
```

Comment:
Are the same tests missing from float/double? It might be nice to add them if so.

Reply:
Apparently we do. 👀